vizro-core/src/vizro/static/css/loading.css

```diff
@@ -0,0 +1,60 @@
+/* Loading spinner */
```

Comment:
This is a pretty cryptic file. It would be nice to explain in a comment:
a. what this is doing (like which loading does it refer to?), since "Loading spinner" could mean lots of different things in Dash.
b. a link to where this weird looking CSS comes from